live-build/base/config/package-lists/minimal.list.chroot

```diff
@@ -43,3 +43,4 @@ open-vm-tools
 openssh-server
 ubuntu-minimal
 auditd
+python2.7
```

Comment:
can you sort this alphabetically, please. "auditd" should be sorted also, but I guess that was missed when it was added.